src/libraries/System.Runtime/System.Runtime.sln

```diff
@@ -1,4 +1,5 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
```

Comment:
Don't commit changes to solution files. They are auto generated and owned by infrastructure.

Reply:
Thanks, I don't known about it. Reverted by https://github.com/dotnet/runtime/pull/115507/commits/1f62c3af5f2215439f77ca59abe9e3b4932ac94a , but current solution does not build from VS without this change.